solutions/swb-reference/README.md

```diff
@@ -6,348 +6,728 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
 
-7. In this root directory run `STAGE=<STAGE> rushx integration-tests`
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: Only Researchers and Project Admins can access this API, the user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Get Connection** request.
+
+GET `{{API_URL}}/projects/:projectId/environments/:id/connections`
+
+In the response you'll find a `url`. Copy and paste that `url` into the browser to view your Sagemaker Notebook instance.
+
+## Stop an Environment
+In **SWBv2 Official** Postman Collection under **environments** folder choose **Stop Environment** API.
+
+In the params tab set `id` parameter to the `ENV_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: The user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Stop Environment** request.
+
+PUT `{{API_URL}}/projects/:projectId/environments/:id/stop`
+
+## Start an Environment
+In **SWBv2 Official** Postman Collection under **environments** folder choose **Start Environment** API.
+
+In the params tab set `id` parameter to the `ENV_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: The user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Start Environment** request.
+
+PUT `{{API_URL}}/projects/:projectId/environments/:id/start`
+
+## Terminate the Environment
+In **SWBv2 Official** Postman Collection under **environments** folder choose **Terminate Environment** API.
+
+In the params tab set `id` parameter to the `ENV_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+In the params tab set `projectId` parameter to the `ENV_PROJECT_ID` value from [Launch Sagemaker Notebook Instance step](#launch-sagemaker-notebook-instance).
+
+Note: The user calling this API needs to have access permissions to the project assigned in the request, for more information see [Assig Project to User](#assign-project-to-user).
+
+Send **Terminate Environment** request.
+
+PUT `{{API_URL}}/projects/:projectId/environments/:id/terminate`
+
+# User Management
```

Comment:
I think this is out of date. Should be ITAdmin not Admin group.

Reply:
I left the instructions for cognito but If we are able to do it by the API I can just remove this, in the mean time I will correct the group name